### PR TITLE
AP-1634 : New changes

### PR DESCRIPTION
**Pull request by ashvini-sib**

### CODE REVIEW

File: docs/AccountApi.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ apiKey.setApiKey("YOUR API KEY");
 
 // Configure API key authorization: partner-key
 ApiKeyAuth partnerKey = (ApiKeyAuth) defaultClient.getAuthentication("partner-key");
-partnerKey.setApiKey("YOUR PARTNER KEY");
+partnerKey.setApiKey("YOUR API KEY");
```
Comment (ekta-slit) [inline docs/AccountApi.md:35]: it should be partner key only

File: src/main/java/sendinblue/ApiClient.java [hunk 2]
```diff
@@ -347,10 +349,7 @@ public void setAccessToken(String accessToken) {
      * @return ApiClient
      */
     public ApiClient setUserAgent(String userAgent) {
-        if (userAgent.toLowerCase().startsWith("sendinblue_"))
-            addDefaultHeader("User-Agent", userAgent);
-        else
-            addDefaultHeader("User-Agent", defaultUserAgent);
+        addDefaultHeader("User-Agent", userAgent);
         return this;
```
Comment (amitsendinblue) [inline src/main/java/sendinblue/ApiClient.java:353]: User agent changes are updated, is this intentional?

File: src/main/java/sendinblue/Json.java [hunk 3]
```diff
@@ -10,356 +10,356 @@
  * Do not edit the class manually.
  */
```
Comment (amitsendinblue) [inline src/main/java/sendinblue/Json.java:12]: It's mentioned in the document that - `Do not change anything in file- src/main/java/sendinblue/Json.java`
Is this intentional?

**Review by amitsendinblue — APPROVED** [review]
LGTM

**Pull request merged**